assets/templates/dataset-filter/range-selector.tmpl

```diff
@@ -35,6 +35,67 @@
+                                    <strong>Year</strong>
+                                    </label>
+                                    <div class="select-alt">
+                                       <select class="select width-sm--10 width-md--10 width-lg--10" name="start-year" id="start-year">
+                                          {{ range $.Data.DateRangeData.YearValues }}
+                                          <option value="{{.}}">{{.}}</option>
+                                          {{ end }}
+                                       </select>
+                                    </div>
+                                 </div>
+                              </div>
+                              <div class="clearfix">
+                                 <div class="margin-bottom--1">
+                                    <span class="font-size-17"><strong>End Date</strong></span>
+                                 </div>
+                                 <div class="col col--md-12 col--lg-12 margin-bottom--2">
+                                    <label class="block margin-bottom--1" for="Month">
+                                    <strong>Month</strong>
+                                    </label>
+                                    <div class="select-alt">
+                                       <select class="select width-sm--10 width-md--10 width-lg--10" name="end-month" id="end-month">
+                                          {{ range $.Data.DateRangeData.MonthValues }}
+                                          <option value="{{.}}">{{.}}</option>
+                                          {{ end }}
+                                       </select>
+                                    </div>
+                                 </div>
+                                 <div class="col col--lg-11 col--md-11 margin-bottom--2">
+                                    <label class="block margin-bottom--1" for="Year">
+                                    <strong>Year</strong>
+                                    </label>
+                                    <div class="select-alt">
+                                       <select class="select width-sm--10 width-md--10 width-lg--10" name="end-year" id="end-year">
+                                          {{ range $.Data.DateRangeData.YearValues }}
+                                          <option value="{{.}}">{{.}}</option>
+                                          {{ end }}
+                                       </select>
+                                    </div>
+                                 </div>
+                                <div class="col col--lg-6 col--md-6 margin-top-md--3 margin-top-lg--3 text-right--lg text-right--md block">
+                                    <input type="submit" class="btn btn--link underline-link padding-top--1" value="Add range">
+                                </div>
+                                 {{else}}
```

Comment:
Incorrect indentation from line 98 to 128; used 1 too many whitespaces for indentation.